typescript-loops-practice/odd-or-even.ts

```diff
@@ -0,0 +1,14 @@
+/* exported oddOrEven */
+function oddOrEven(numbers: number[]): string[] {
+  const newArray = [];
+  let checker;
+  for (let i = 0; i < numbers.length; i++) {
+    checker = numbers[i] % 2;
+    if (checker === 0) {
+      newArray.push('even');
+    } else {
+      newArray.push('odd');
+    }
+  }
+  return newArray;
+}
```

Comment:
No change necessary here but you could make this less lines of code that is still easy to read by having your condition in your `if` statement.

`if (numbers[i] % 2 === 0)`